e2e/specs/conditions.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { test } from '../core';
+import { ConditionsPage } from '../pages';
+import { expect } from '@playwright/test';
+import { generateRandomPatient, deletePatient, Patient } from '../commands';
+
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+    patient = await generateRandomPatient(api);
+  });
+
+test('Should add, delete and edit a condition', async ({page, api}) => {
```

Comment:
Hey @jayasanka-sack, @Piumal1999 and @RandilaP! I think it's a good idea to write separate tests here. By doing so, we can run them one after the other. Since we're mimicking the user, these separate flows will help us better understand how things work instead of handling them as a single unit. WDYT?

Reply:
I tried what you suggested. But it didn't work as I expected